lib/presentation/screens/goal_screen/goal_layout.dart

```diff
@@ -16,35 +17,32 @@ import 'package:water_tracker/presentation/widgets/title_settings_widget.dart';
 class GoalLayout extends StatefulWidget {
   const GoalLayout({Key? key}) : super(key: key);
 
+  @override
+  State<GoalLayout> createState() => _GoalLayoutState();
+}
+
+class _GoalLayoutState extends State<GoalLayout> {
+  var goals = Goal.drinkWater;
+  List<Goal> goalsList = [];
```

Comment:
Why is it here? And i don't see that it should work

Reply:
it works i checked

Reply:
Goals variable not needed